### ISSUE TITLE
[MEDIUM] Consistency model includes target network params in optimizer

### ISSUE THREAD

**Issue by Maxelee** [Maxelee/vdm_BIND#20]
## Description

In `vdm/consistency_model.py`, the `configure_optimizers()` method passes `self.parameters()` to the optimizer, which includes **both** the online network and the target network parameters. The target network should be updated via EMA only, not by gradient descent.

## Current Code

```python
def configure_optimizers(self):
    optimizer = torch.optim.Adam(self.parameters(), lr=self.lr)
    return optimizer
```

`self.parameters()` includes both `self.network` and `self.target_network`.

## Impact

- Target network receives gradient updates, violating the consistency model algorithm
- The target is supposed to be a slow-moving average of the online network
- Training may still converge but will be suboptimal and theoretically incorrect
- Wastes GPU memory storing gradients for target parameters

## Fix

Only optimize the online network parameters:

```python
def configure_optimizers(self):
    optimizer = torch.optim.Adam(self.network.parameters(), lr=self.lr)
    return optimizer
```

And ensure target network params have `requires_grad=False`:
```python
for p in self.target_network.parameters():
    p.requires_grad_(False)
```

## Files
`vdm/consistency_model.py` — `configure_optimizers()` and constructor
